style.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;600;700;800;900&display=swap');
+
+:root {
+    --primary: #2489bb;
+    --secondary: #111212;
+    --secondary-light: #1a1b1b;
+    --accent: #FFEA29;
+    --secondary-grey: #f7f7f7;
+    --accent-light: #726d3c;
+    --font-primary: #f1f1f1;
+    --font-secondary: #b4b3b3;
+    --btn-primary: #2489bb;
+    --btn-primary-hover: #017DBC;
+    --white: #ffffff;
+    --card-shadow:0px 0px 6px 1px #cdc9c9;
+}
+
+* {
+    font-size: 16px;
+    font-family: 'Rubik', sans-serif;
+    margin: 0;
+    letter-spacing: 1.5px;
+    box-sizing: border-box;
+}
+
+.nav {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
You can declare these properties in util and reuse this as class in multiple areas